ci/Jenkinsfile_docker_cache

```diff
@@ -22,34 +22,18 @@
 
 // timeout in minutes
 total_timeout = 300
-git_timeout = 15
-// assign any caught errors here
-err = null
 
-// initialize source codes
-def init_git() {
-  deleteDir()
-  retry(5) {
-    try {
-      // Make sure wait long enough for api.github.com request quota. Important: Don't increase the amount of
-      // retries as this will increase the amount of requests and worsen the throttling
-      timeout(time: git_timeout, unit: 'MINUTES') {
-        checkout scm
-        sh 'git submodule update --init --recursive'
-        sh 'git clean -x -d -f'
-      }
-    } catch (exc) {
-      deleteDir()
-      error "Failed to fetch source codes with ${exc}"
-      sleep 2
-    }
-  }
+node('restricted-mxnetlinux-cpu') {
```

Comment:
this is inconsistent with the constant labels style from the other files, why?

Reply:
Good catch, I missed replacing it

Reply:
Ah, it's explained below: 

```
// Loading the utilities requires a node context unfortunately
```

We can not use the constants at that point because the utility to load a constant has not been loaded at that point.

Reply:
Do we need a node to define some constants? Could we have a separate file for them and load it before?

Reply:
Unfortunately yes. It's not possible to load other groovy scripts without being in a node context